packages/sysweb3-utils/src/transactions.ts

```diff
@@ -39,21 +39,10 @@ export const txUtils = () => {
 
   const getFeeRate = (fee: number): BigInt => new sys.utils.BN(fee * 1e8);
 
-  /**
```

Comment:
Why remove comments?

Reply:
comments are unnecessary since the name of the methods are already describing what they do, as well as the return type and params